### DISCUSSION TITLE
Will Newpipe expose more YouTube video and audio quality options?

### DISCUSSION THREAD

**Discussion by Booms1981** [TeamNewPipe/NewPipe#10338] [Questions]
Using Seal I can see AV1 encoded video, and many more audio bitrates.

Will these be made available in Newpipe? I like Seal, but I can't add several videos to the download manager at once.

**Comment by opusforlife2** [comment]
Yes, eventually Newpipe should be able to support more resolutions. But that will take time. See the pinned discussion on the rewrite.
